### ISSUE TITLE
FontResizer Matlab 2014b

### ISSUE THREAD

**Issue by lmanubens** [MarcelBeining/T2N#4]
Hi Marcel,

In t2n_Tutorial.m, when calling t2n_FIplot (lines 50 and 71) and t2n_plotbAP (lines 78 and 197) the function FontResizer is not found. 

I'm using Matlab 2014b because it's the only distribution I can use with my institution's license. By commenting those lines out all the tutorial code runs smoothly :)

Best!

**Comment by MarcelBeining** [comment]
Hi Linus,
though I apologize for having problems with T2N, I like your motivation to go through it all and to dig into problems before reporting them :-)
Concerning your issue, I guess if you go to the T2N folder under `src/masterplotter` that folder (which should contain the functions that are missing) is empty, right? This is because it is a git submodule of T2N and unfortunately git submodules have to be fetched separately when cloning a repository.
Use a command line terminal and cd to the T2N main folder. Then do
`git submodule update --init --recursive`
Afterwards the src/masterplotter folder should not be empty anymore.
If you do not have any git software installed, you need to install one first (e.g. https://gitforwindows.org/ for Windows, make sure to activate the checkbox saying to add git to the Windows PATH such that the git command is found from the command line)

I have included a note in the Documentation and found a way to initialize the submodules programmatically when running the t2n_runthisAfterUnzip.m file after installation.

**Comment by MarcelBeining** [comment]
I assume that the issue is solved, otherwise please reopen it.

**Comment by neuroptics** [comment]
I am also having this issue, following the above instructions produces an error: 
"fatal: not a git repository (or any of the parent directories): .git"

I tried running git using t2n_runthisAfterUnzip.m and also from the command line, both produce the same error. The main T2N folder contains ".gitattributes", ".gitignore", and ".gitmodules", but no ".git" files. Not sure what is missing.

I added masterplotter manually and it fixed the issue described above.
